hat/assets/js/apps/Iaso/domains/orgUnits/components/OrgUnitMultiReferenceInstances.tsx

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable camelcase */
+import React, { FunctionComponent, useState } from 'react';
+import { Box, Grid } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+import {
+    IconButton as IconButtonComponent,
+    useSafeIntl,
+} from 'bluesquare-components';
+
+import { Link } from 'react-router';
+import { baseUrls } from '../../../constants/urls';
+import InputComponent from '../../../components/forms/InputComponent';
+import InstanceFileContent from '../../instances/components/InstanceFileContent';
+import MESSAGES from '../messages';
+import WidgetPaper from '../../../components/papers/WidgetPaperComponent';
+import { Instance } from '../../instances/types/instance';
+
+type Props = {
+    referenceInstances: Instance[];
+};
+
+const useStyles = makeStyles(() => ({
+    formContents: {
+        width: '100%',
+    },
+    link: {
+        textDecoration: 'underline',
+        '&:active': { textDecoration: 'none' },
+        '&:hover': { textDecoration: 'none !important' },
+    },
+}));
+
+export const OrgUnitMultiReferenceInstances: FunctionComponent<Props> = ({
+    referenceInstances,
+}) => {
+    const classes = useStyles();
+    const { formatMessage } = useSafeIntl();
+
+    const [active, setActive] = useState<Instance>(referenceInstances[0]);
+
+    const options = referenceInstances.map(instance => ({
+        value: instance.id,
+        label: instance.form_name,
+    }));
+    const url = `${baseUrls.instanceDetail}/instanceId/${active.id}/referenceFormId/${active.form_id}`;
+    const title = `${formatMessage(MESSAGES.detailTitle)} - ${
+        active.form_name
+    }`;
```

Comment:
Re-renders aren't a big deal in most cases. I don't think that a simple CPU bound operation could lead to performance problems here.

Reply:
useMemo is not required while getting a string or a boolean FYI

Reply:
what about the `options` variable?
Also, could you explain why it's not needed for strings and booleans?
It seems to me that `url` and `title` are both computing strings, which can be costly.

Reply:
> In my personal opinion, it would be a waste of time to wrap every single object/array/function in these hooks. The benefits are negligible in most cases; React is highly optimized, and re-renders often aren't as slow or expensive as we often think they are!

https://www.joshwcomeau.com/react/usememo-and-usecallback/#when-to-use-these-hooks-6